packages/info/README.md

```diff
@@ -54,6 +54,29 @@ module.exports = class FooMixin extends Mixin {
 };
 ```
 
+### `getRuntimeLogger()` ([callable](https://github.com/untool/mixinable/blob/master/README.md#defineoverride))
+
+This utility hook defined by `hops-info` provides other mixins with a fully configured logger instance. This logger has three standard log methods (`info`, `warning` and `error`) that correspond to the lower three log levels described above. Additionally, it supports arbitrary log methods (e.g. `request`, `hint`, `foo`).
+
+```javascript
+const { Mixin } = require('hops-mixin');
+
+module.exports = class BarRuntimeMixin extends Mixin {
+  foo() {
+    if (typeof this.getLogger === 'function') {
+      const logger = this.getLogger();
```

Comment:
```suggestion
    if (typeof this.getRuntimeLogger === 'function') {
      const logger = this.getRuntimeLogger();
```
Btw, why do we need to check if `this.getLogger` is a function?

Reply:
That's a very good question!

Reply:
Because earlier not all apps had the `@untool/info` package installed.

Reply:
And I guess also sometimes you call a hook before the logger is initialized.